docs/features/logforwarding/outputs/splunk-forwarding.md

```diff
@@ -0,0 +1,66 @@
+
+### Steps to forward to Splunk HTTP Event Collector (HEC)
```

Comment:
We should also consider defining the splunk configuration we used to make all this work.

Reply:
Do you mean the vector sink?  
----
[sinks.splunk_hec]
type = "splunk_hec"
inputs = ["my-logs"]
endpoint = "https://example-splunk-hec-service:8088/endpoint"
compression = "none"
default_token = "VS0BNth3wCGF0hecToken234"
encoding.codec = "json"
----

Reply:
Per our slack conversation referring to the receiver config, we do have existing tests that cover some basic logging, but they may be too simplistic for docs at this time.   We will plan to add to this document, once we have some feedback from support or QE.